src/utils/react-live-utils.js

```diff
@@ -0,0 +1,83 @@
+
```

Comment:
A lot of code here seems old school ES3. Could you make use of newest JS features such as arrow functions and so on?